### PR TITLE
GitHub templates

### PR DESCRIPTION
**Pull request by dieti80**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Issue Request Template
+
+## Description
+Please include a summary of the request.
+
+## Motivation
+Please include a motivation for this issue request.
+
+## Type of request
```
Comment (jbe99) [inline .github/issue_template.md]: This can be replaced by the labels.

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Please describe what has been observed.
+
+### Expected behavior
+Please describe which behavior was expected.
+
+### Current behavior
+Please describe which behavior has been observed.
+
+### Additional information
+- [ ] When did this behavior first occur? What has changed?
```
Comment (jbe99) [inline .github/issue_template.md]: I would remove the checkbox here.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- [ ] Bug fix (non-breaking change)
+- [ ] Bug fix (breaking change, which have impact to existing functionality)
+- [ ] Feature (non-breaking change)
+- [ ] Feature (breaking change, which have impact to existing functionality)
+- [ ] Documentation
+
+## How has this been tested?
+Please describe the tests you ran to verify the changes. Provide instructions that we can re-run the tests. Please list any relevant details for tests setup.
+
+- [ ] Test A
```
Comment (jbe99) [inline .github/pull_request_template.md]: I would remove the checkbox here.

File: .github/issue_template.md (hunk 2 above)
Comment (dieti80) [reply .github/issue_template.md]: Done

File: .github/pull_request_template.md (hunk 3 above)
Comment (dieti80) [reply .github/pull_request_template.md]: Done

**Review by jbe99 — APPROVED** [review]

File: .github/issue_template.md (hunk 1 above)
Comment (dieti80) [reply .github/issue_template.md]: Done

**Pull request merged**